### DISCUSSION TITLE
Seeking assistance. Requesting resources and identifying good first issues for contributions.

### DISCUSSION THREAD

**Discussion by vinayakjeet** [cncf/mentoring#1109] [Q&A] [answered]
Hello, I am a 2nd-year student at BITS Pilani with a keen interest in open source. I am eager to contribute to the CNCF (Cloud Native Computing Foundation) community. Could you please provide resources to help me get started, as well as suggest some good first issues for a beginner like me? Your guidance will be greatly appreciated. Thank you!

**Comment by nate-double-u** [comment, marked as answer]
Hi @vinayakjeet!

There's a couple good resources for getting started, one I like to share is an article @Riaankl wrote for just such a question: "[Do you want to start contributing to open source and need help figuring out where to begin?](https://contribute.cncf.io/contributors/getting-started/)" It's a great overview of how to find a project and get going on helping out.

We also have a tool we've been developing (currently in alpha) to help folks find projects looking for contributors: [CLOTributor](https://clotributor.dev/). It's useful for helping contributors find projects that suit their skills and interests best.
